docs/occ.md

```diff
@@ -0,0 +1,938 @@
+# Talk occ commands
+
+ * [talk:command:add](#talkcommandadd)
+ * [talk:command:add-samples](#talkcommandadd-samples)
+ * [talk:command:delete](#talkcommanddelete)
+ * [talk:command:list](#talkcommandlist)
+ * [talk:command:update](#talkcommandupdate)
+ * [talk:developer:update-docs](#talkdeveloperupdate-docs)
+ * [talk:monitor:calls](#talkmonitorcalls)
+ * [talk:active-calls](#talkactive-calls)
+ * [talk:monitor:room](#talkmonitorroom)
+ * [talk:room:add](#talkroomadd)
+ * [talk:room:create](#talkroomcreate)
+ * [talk:room:delete](#talkroomdelete)
+ * [talk:room:demote](#talkroomdemote)
+ * [talk:room:promote](#talkroompromote)
+ * [talk:room:remove](#talkroomremove)
+ * [talk:room:update](#talkroomupdate)
+ * [talk:signaling:add](#talksignalingadd)
+ * [talk:signaling:delete](#talksignalingdelete)
+ * [talk:signaling:list](#talksignalinglist)
+ * [talk:stun:add](#talkstunadd)
+ * [talk:stun:delete](#talkstundelete)
+ * [talk:stun:list](#talkstunlist)
+ * [talk:turn:add](#talkturnadd)
+ * [talk:turn:delete](#talkturndelete)
+ * [talk:turn:list](#talkturnlist)
+ * [talk:user:remove](#talkuserremove)
+ * [talk:user:transfer-ownership](#talkusertransfer-ownership)
+## talk:command:add
+
+Add a new command
+
+### Usage
+
+* `talk:command:add <cmd> <name> <script> <response> <enabled>`
+
+### Arguments
+
+#### `cmd`
+
+The command as used in the chat "/help" => "help"
+
+* Is required: yes
+* Is array: no
+* Default: `NULL`
+
+#### `name`
+
+Name of the user posting the response
+
+* Is required: yes
+* Is array: no
+* Default: `NULL`
+
+#### `script`
+
+Script to execute (Must be using absolute paths only)
+
+* Is required: yes
+* Is array: no
+* Default: `NULL`
+
+#### `response`
+
+Who should see the response: 0 - No one, 1 - User, 2 - All
+
+* Is required: yes
+* Is array: no
+* Default: `NULL`
+
+#### `enabled`
+
+Who can use this command: 0 - Disabled, 1 - Moderators, 2 - Users, 3 - Guests
+
+* Is required: yes
+* Is array: no
+* Default: `NULL`
```

Comment:
I think the layout should be a table and not a h2-h4 list.
Let's have a table for the arguments and one for the options similar to the output of `--help`
Otherwise every command is more than a 4k screen in size and that's not helpful anymore

Reply:
>  Otherwise every command is more than a 4k screen in size and that's not helpful anymore

I will check what I can do to change the layout

Reply:
@nickvergessen look this:

![Screenshot_20230302_124852](https://user-images.githubusercontent.com/1079143/222478567-c2f299d0-7a58-4ee3-9630-0cba62226e55.png)

Reply:
I just didn't format the markdown with the spaces because this will be a complex logic or, maybe, will require a new composer dependency to do this.

Reply:
Looks much better!